reflex/utils/hosting.py

```diff
@@ -736,7 +753,9 @@ def convert_to_local_time(iso_timestamp: str) -> str:
         The converted timestamp string.
     """
     try:
-        local_dt = datetime.fromisoformat(iso_timestamp).astimezone()
+        local_dt = convert_to_local_time_with_tz(iso_timestamp)
+        if local_dt is None:
+            return iso_timestamp
         return local_dt.strftime("%Y-%m-%d %H:%M:%S.%f %Z")
     except Exception as ex:
```

Comment:
Same here, can we catch a more specific error / do we know why we need the catch?

Reply:
Actually we don't need to catch here, it's already caught in the call to convert.